Taskfile.yml

```diff
@@ -232,6 +232,10 @@ tasks:
         -postgresql-url='postgres://username:password@127.0.0.1:5432/postgres'
         -compat-url='mongodb://username:password@127.0.0.1:47017/?replicaSet=rs0'
         -no-xfail={{.NO_XFAIL}}
+      - >
+        ../bin/go-junit-report -parser=gojson -iocopy
+        -in postgresql-{{.SHARD_INDEX}}-{{.SHARD_TOTAL}}-stdout.json.tmp
+        -out postgresql-{{.SHARD_INDEX}}-{{.SHARD_TOTAL}}.xml
```

Comment:
I don't think this works if we use `task test-integration-postgresql RUN_TEST=xxx`, which is a common usage for local development.

Reply:
It works, the default `SHARD_INDEX` and `SHARD_TOTAL` is zero, so the file `postgresql-0-0-stdout.json.tmp` is created.

Reply:
> Any reason why we do it only for postgresql?

My mistake :sweat_smile: